lib/service/Service.ts

```diff
@@ -358,8 +358,12 @@ class Service {
   /**
    * Get a map between pair ids and its orders from the order book.
    */
-  public listOrders = (args: { pairId: string, includeOwnOrders: boolean, limit: number }): Map<string, OrderSidesArrays<any>> => {
-    const { pairId, includeOwnOrders, limit } = args;
+  public listOrders = (
+    args: { pairId: string, owner: Owner | number, limit: number },
+    ): Map<string, OrderSidesArrays<any>> => {
+    const { pairId, owner, limit } = args;
+    const includeOwnOrders = owner !== Owner.Peer;
+    const includePeerOrders = owner !== Owner.Own;
```

Comment:
I don't know if we'll ever add or change the options of Own/Peer/Both, but just in case it might be good to have this logic check "if Peer or Both" for includePeerOrders and "if Own or Both" for includeOwnOrders.

Reply:
Alright, sounds reasonable, made the changes :)